From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Peter Johnson <johnson.peter@gmail.com>
Date: Fri, 10 Oct 2014 09:15:42 -0700
Subject: [PATCH 0404/6262] Counter::setUpSource(AnalogTrigger): actually set
 the source.

Previously this created the output but didn't set up the source.

Change-Id: Ifac7ed589358e15bc70ba46ae4a756ce56683a8a
---
 .../src/main/java/edu/wpi/first/wpilibj/Counter.java            | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wpilibj/wpilibJavaDevices/src/main/java/edu/wpi/first/wpilibj/Counter.java b/wpilibj/wpilibJavaDevices/src/main/java/edu/wpi/first/wpilibj/Counter.java
index d6550ea23c22a22a8d187561d2b6d2daa3ca10c5..07d676a9a8bb74324f75e90e92d81e81c884a94d 100644
--- a/wpilibj/wpilibJavaDevices/src/main/java/edu/wpi/first/wpilibj/Counter.java
+++ b/wpilibj/wpilibJavaDevices/src/main/java/edu/wpi/first/wpilibj/Counter.java
@@ -257,7 +257,7 @@ public class Counter extends SensorBase implements CounterBase,
 		if (triggerType == null){
 			throw new NullPointerException("Analog Trigger Type given was null");
 		}
-		analogTrigger.createOutput(triggerType);
+		setUpSource(analogTrigger.createOutput(triggerType));
 		m_allocatedUpSource = true;
 	}
 
